resources/frequently-asked-questions/README.md

```diff
@@ -24,7 +24,7 @@ There are several new features and benefits of using the Experience Platform Mob
 | Adobe Campaign Standard | Push and in-app messaging | Push only |
 | Adobe Target | ✔️ | ✔️ |
 | Places Service | ✔️ |  |
-| Project Griffon \(BETA\) | ✔️ |  |
+| Adobe Experience Platform Assurance \(BETA\) | ✔️ |  |
```

Comment:
Take the "BETA" off of this as well :) .

Reply:
Done! Beta removed.